### DISCUSSION TITLE
Plot the deformation field

### DISCUSSION THREAD

**Discussion by Uemerson** [voxelmorph/voxelmorph#528] [Q&A] [answered]
How can I plot the deformation field in the same way as the images below:

![Screenshot_20230613_124341](https://github.com/voxelmorph/voxelmorph/assets/19247745/3321bfc0-78d1-4c14-bca4-a9990fe14351)

Is there any example? Can I do this with the lib neurite?
What terms do I look for to be able to reproduce a similar result?

**Comment by adalca** [comment, marked as answer]
Sure, the visualization on the left is simply a RGB image of a slice of the 3D field, so plt.imshow should work

For the ones on the right, you can do:

- get a bw grid from `pystrum.pynd.ndutils.bw_grid`
- deform it with the tools in voxelmorph (e.g. vxm.networks.Transform() )
- visualize the resulting grayscale image

**Reply by Uemerson** [reply]
Thanks for the quick response, I'll follow the steps. Thanks again!

**Comment by adalca** [comment]
I made a quick tutorial: https://colab.research.google.com/drive/1F8f1imh5WfyBv-crllfeJBFY16-KHl9c?usp=sharing

**Reply by Uemerson** [reply]
Thank you very much, I will compare with what I had done :)

About the RGB, what color palettes did you use to match the images I posted above ?

**Comment by adalca** [comment]
oh, sorry that I couldn't tell you :( Been a tad too long

**Reply by Uemerson** [reply]
Ok, thanks for helping 💚
